webapp/components/test-file-results-table.js

```diff
@@ -133,22 +133,22 @@ class TestFileResultsTable extends WPTColors(PathInfo(TestRunsBase)) {
           </td>
         </template>
       </tr>
+    </template>
 
-      <template is="dom-if" if="[[verbose]]">
-        <template is="dom-if" if="[[anyScreenshots(row)]]">
-          <tr>
-            <td class="sub-test-name"><code>Screenshot</code></td>
-            <template is="dom-repeat" items="[[row.results]]" as="result">
-              <td>
-                <template is="dom-if" if="[[ testScreenshot(result.screenshots) ]]">
-                  <a href="[[ computeAnalyzerURL(result.screenshots) ]]">
-                    <img src="[[ testScreenshot(result.screenshots) ]]" />
-                  </a>
-                </template>
-              </td>
-            </template>
-          </tr>
-        </template>
+    <template is="dom-if" if="[[verbose]]">
```

Comment:
This is simply moving the whole dom-if out of the `<template is="dom-repeat">` - we were producing an extra dom-if per row, when this dom-if only applies to screenshot (which have just one row). That meant that the css `nth-child(2)` selector was hitting the dom-if, not the 2nd `tr`.